RELEASE.md

```diff
@@ -4,12 +4,16 @@ The releases will be done through [Release Please]. This project uses the [Githu
 
 All configuration files for [Release Please] are found in the `./ci` directory.
 
+### Pub Dev
+
+The pub dev release is done by running the `publish_pub_dev` Github action.
```

Comment:
Hi Nico, I just happened to come across this pr. It could be that triggering a release with `workflow_dispatch` will be blocked by pub.dev.

> Pub.dev only allows automated publishing from GitHub Actions when the workflow is triggered by pushing a git tag to GitHub. Pub.dev rejects publishing from GitHub Actions triggered without a tag. This ensures that new versions cannot be published by events that should never trigger publishing.

See https://dart.dev/tools/pub/automated-publishing#publishing-packages-using-github-actions

P.S. Awesome work!

Reply:
Hi @josxha 

glad to have you peeking in here ;) 
and have you review some of my work (I really do mean that!). Maybe you now understand some more of my comments in previous issues (e.g. https://github.com/maplibre/flutter-maplibre-gl/issues/436). And you might understand why I am very much excited for the iOS part of https://github.com/josxha/flutter-maplibre !

There's a couple of applications using this package by now :)

To your comment. Yes, I need to improve upon the CI / **CD** part for sure. I found this for the workflow_dispatch trigger type and publishing to pub dev: https://github.com/dart-lang/pub-dev/issues/7177#issuecomment-2306713948

So I guess I will change this action to checkout a tag ref first (the one selected). The other option is that I directly trigger the
```
on:
  push:
    tags:
```
style, but having another Github App to do the Release Please part. Otherwise, the merge of the outstanding PRs from Release Please will not do anything (not triggering subsequent workflows).

Reply:
And one more remark: all of this is only possible because some random people did some things in their free time 😉 
so the real credits of SBB should go towards **small** OpenSource projects and contributors (like you!)